keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -931,28 +716,27 @@ Several components must be implemented or modified in Kubernetes:
   ResourceClaim (directly or through a template) and ensure that the
   resource is allocated before the Pod gets scheduled, similar to
   https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/scheduling/scheduler_binder.go
-- Kubelet must be extended to retrieve information from ResourceClaims
-  and to call a resource kubelet plugin. That plugin returns CDI device ID(s)
+- Kubelet must be extended to manage ResourceClaims
+  and to call a kubelet plugin. That plugin returns CDI device ID(s)
   which then must be passed to the container runtime.
 
-A resource driver can have the following components:
-- *CRD controller* (optional): a central component which translates parameters
-  defined with a vendor CRD into in-tree parameter types. 
-- *kubelet plugin* (required): a component which cooperates with kubelet to
-  publish resource information and to prepare the usage of the resource on a node.
+A DRA driver can have the following components:
+- *admission webhook* (optional): a central component which checks the opaque
+  configuration parameters in ResourceClaims, ResourceClaimTemplates and DeviceClasses at
+  the time that they are created. Without this, invalid parameters can only
+  be detected when a Pod is about to run on a node.
```

Comment:
So this will allow us to return an error at claim creation time and block it from being created if the CRD embedded in the config is not valid?

Reply:
Yes.